subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -824,9 +824,42 @@ config BT_CTLR_RX_PDU_META
 	prompt "RX pdu meta data"
 	bool
 
+config BT_CTLR_NRF_GRTC
```

Comment:
Wouldn't it be better to have this type of settings in a new LLL specific kconfig file? (also the other nordic specific ones which are now in this file)

Reply:
yes, a future PR to move nRF specific to `Kconfig.ll_sw_split.nrf`, `Kconfig.ll_sw_split.openisa` etc...